{"name": "Негативный тест авторизации", "status": "passed", "steps": [{"name": "Шаг 1. Вводим ложные логин/пароль", "status": "passed", "start": 1738134576096, "stop": 1738134576114}], "parameters": [{"name": "browser_name", "value": "'chromium'"}, {"name": "username", "value": "'admin'"}, {"name": "password", "value": "'nhy'"}, {"name": "result", "value": "'Invalid credentials. Please try again.'"}], "start": 1738134571407, "stop": 1738134576114, "uuid": "909e7ab5-b8fa-4f4f-941b-f304950e8468", "historyId": "fec8b3fd9186f48405c031b7359fd62f", "testCaseId": "68772be64bb38f6a4c2b59109d7c35bc", "fullName": "test_login#test_login_failure", "labels": [{"name": "story", "value": "Login Feature"}, {"name": "severity", "value": "normal"}, {"name": "suite", "value": "test_login"}, {"name": "host", "value": "DESKTOP-TR3R9M3"}, {"name": "thread", "value": "45524-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_login"}]}